README.md

```diff
@@ -135,6 +135,8 @@ Each time a new commit is pushed to the pull request, the `Approved` label will
 
 After the CRDA scan is approved and the workflow runs, a label indicating the scan result will be added to the pull request.
 
+> **Note**: Pull requests created by repository maintainers will automatically have `CRDA Scan Approved` label.
```

Comment:
let's not use "maintainers" since it doesn't appear to be a GitHub term

we can simply say, "Pull requests authored by users with write access to the repository will automatically receive the `CRDA Scan Approved` label."